conda_smithy/configure_feedstock.py

```diff
@@ -89,7 +89,7 @@ def compute_build_matrix(meta):
 
 def main(forge_file_directory):
     recipe_dir = 'recipe'
-    config = {'docker': {'image': 'pelson/obvious-ci:latest_x64', 'command': 'bash'},
+    config = {'docker': {'image': 'jjhelmus/conda-forge-x86_64:latest', 'command': 'bash'},
```

Comment:
This should be possible. I made a condaforge organization on dockerhub (it did not like the hyphen) and added everyone as owners.  Network issues are holding be back from pushing the image up the the organization but it should be there later today.